### ISSUE TITLE
PHP error: Unable to prepare query

### ISSUE THREAD

**Issue by Systems-Gz** [osTicket/osTicket#5984]
* [ ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [X] Do you have any errors in the PHP error log, or javascript console?
* [X] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [X] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Description

Searching users on the "new ticket" menu does not display the dropdown menu with suggested users, throwing this error:
`[Thu Oct 07 11:45:10.224452 2021] [php7:error] [pid 6349] [client 10.255.0.246:57976] PHP Fatal error:  Uncaught InconsistentModelException: Unable to prepare query:  SELECT A1.`id`, A1.`name`, A6.`address` FROM `ost_user` A1 LEFT JOIN `ost_user_email` A2 ON (A1.`id` = A2.`user_id`) LEFT JOIN `ost_organization` A3 ON (A1.`org_id` = A3.`id`) LEFT JOIN `ost_user_account` A4 ON (A1.`id` = A4.`user_id`) LEFT JOIN `ost_user__cdata` A5 ON (A1.`id` = A5.`user_id`) LEFT JOIN `ost_user_email` A6 ON (A1.`default_email_id` = A6.`id`) WHERE (A2.`address` LIKE '%albert%' OR A1.`name` LIKE '%albert%' OR A3.`name` LIKE '%albert%' OR A4.`username` LIKE '%albert%' OR A5.`phone` LIKE '%albert%') LIMIT 25 in /var/www/html/replaced.domain.com/v1.15/upload/include/class.orm.php:3467\nStack trace:\n#0 /var/www/html/replaced.domain.com/v1.15/upload/include/class.orm.php(3522): MySqlExecutor->execute()\n#1 /var/www/html/replaced.domain.com/v1.15/upload/include/class.orm.php(2070): MySqlExecutor->getRow()\n#2 /var/www/html/replaced.domain.com/v1.15/upload/include/class.orm.php(2043): FlatArrayIterator->{closure}()\n#3 /var/www/html/ in /var/www/html/replaced.domain.com/v1.15/upload/include/class.orm.php on line 3467, referer: https://replaced.domain.com/scp/tickets.php?a=open`

I have other osTicket instances running on the same machine, same osTicket version and it works fine.

### Steps to Reproduce

1. Click on create new ticket
2. Try and search by name/tlf/email
3. The dropdown suggesting users based on the input is not displayed.

### Versions

osTicket v1.15.3.1
debian 5.10.46

**Comment by JediKev** [comment]
@Systems-Gz 

It's not a full error so I can't really say but I have a feeling it has something to do with User CDATA table. Just fully drop/delete your `ost_user__cdata` table and either run cron manually via cli - or - add a dummy field to Contact Information Form, Save Changes, and delete it. This should regenerate the CDATA table for Users and hopefully resolve that error.

Cheers.

**Comment by Systems-Gz** [comment]
@JediKev I dropped the table and now it doesn't throw errors anymore, but the dropdown menu is still not showing.

**Comment by JediKev** [comment]
@Systems-Gz 

Then you will need to do some debugging to figure out where the issue lies. Without errors it's near impossible for us to see what's causing the issue.

Cheers.

**Comment by Systems-Gz** [comment]
@JediKev 
I'm at a loss right now, what I see is that I'm getting an empty response packet. I logged the querys to see if something is wrong with them.

Query from the problematic instance:
`SELECT A1.`id`, A1.`name`, A6.`address` FROM `ost_user` A1 LEFT JOIN `ost_user_email` A2 ON (A1.`id` = A2.`user_id`) LEFT JOIN `ost_organization` A3 ON (A1.`org_id` = A3.`id`) LEFT JOIN `ost_user_account` A4 ON (A1.`id` = A4.`user_id`) LEFT JOIN `ost_user__cdata` A5 ON (A1.`id` = A5.`user_id`) LEFT JOIN `ost_user_email` A6 ON (A1.`default_email_id` = A6.`id`) WHERE (A2.`address` LIKE '%alberto%' OR A1.`name` LIKE '%alberto%' OR A3.`name` LIKE '%alberto%' OR A4.`username` LIKE '%alberto%' OR A5.`phone` LIKE '%alberto%') LIMIT 25`

Query from a working instance:

`SELECT A1.`id`, A1.`name`, A5.`address` FROM `ost_user` A1 LEFT JOIN `ost_user_email` A2 ON (A1.`id` = A2.`user_id`) LEFT JOIN `ost_organization` A3 ON (A1.`org_id` = A3.`id`) LEFT JOIN `ost_user_account` A4 ON (A1.`id` = A4.`user_id`) LEFT JOIN `ost_user_email` A5 ON (A1.`default_email_id` = A5.`id`) WHERE (A2.`address` LIKE '%alberto%' OR A1.`name` LIKE '%alberto%' OR A3.`name` LIKE '%alberto%' OR A4.`username` LIKE '%alberto%') LIMIT 25`

I see that they're different but I can't point at something being wrong. I hope you can get something from this info.

**Comment by JediKev** [comment]
@Systems-Gz 

One is joining the `ost_user__cdata` table and the other is not.

Cheers.

**Comment by JediKev** [comment]
@Systems-Gz 

It only does this if you have the phone field on the contact information form. 
- https://github.com/osTicket/osTicket/blob/develop/include/ajax.users.php#L83-L84

Regardless, it shouldn’t matter as it’s an OR statement so it is conditional.

Cheers.

**Comment by Systems-Gz** [comment]
Dropping `ost_user__cdata` DID fix the problem, I was testing the fix incorrectly.
